publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -252,8 +254,33 @@ The pipelines can be associated with an application stack in the Kabanero custom
             url: https://github.com/kabanero-io/kabanero-pipelines/releases/download/0.9.0/default-kabanero-pipelines.tar.gz
     ```
 
-    When the product operator activates the CRD, it associates the pipelines in the pipelines archive with each of the stacks in the stack hub. The default pipelines are intended to work with all the stacks in the stack hub in the previous example. All of the pipeline-related resources (such as the tasks, trigger bindings, and pipelines) prefix the name of the resource with the keyword `StackId`. When the operator activates these resources, it replaces the keyword with the name of the stack it is activating.
+* Experimental gitops pipelines (Tech Preview)
+
+    The experimental GitOps pipelines are intended to be used when you want to use Tekton Webhooks to drive GitOps actions in pipelines.  They are not associated with the stacks unlike the other two set of pipeline. They are scoped to the Kabanero instance and are meant to build, promote, and deploy all the default stacks.  These are meant to specified under the `gitops / pipelines` section of the Kabanero CR YAML.  These are not activated by default and will have to be added to your Kabanero CR.
```

Comment:
Let's rewrite as:
You can use the experimental GitOps pipelines when you want to use Tekton Webhooks to drive GitOps actions in pipelines. Unlike the other pipelines, the GitOps pipelines are not associated with individual stacks, but are instead scoped to the instance in order to build, promote, and deploy all the stacks. Because they are scoped to the instance, they must be specified in the Kabanero CR, in the `gitops / pipelines` section.